python/pyspark/sql/catalog.py

```diff
@@ -79,6 +84,42 @@ def __init__(self, sparkSession: SparkSession) -> None:
         self._jsparkSession = sparkSession._jsparkSession
         self._jcatalog = sparkSession._jsparkSession.catalog()
 
+    def currentCatalog(self) -> str:
```

Comment:
Just dropping a comment .. let's make sure these are seen in PySpark documentation properly.

Reply:
let me check it.